pyproject.toml

```
@@ -1,15 +1,86 @@
 [build-system]
 requires = [
-    "setuptools>=66.1.0; python_version>='3.12'",
-    "setuptools>=60.0.0; sys.platform=='win32'",
-    "setuptools<v60.0; python_version<'3.12' and sys.platform!='win32'",
+    "setuptools>=61.0.0",
+    "setuptools_scm<9",
```

If you were to drop 3.7 it would look like this:
```suggestion
    "setuptools>=64",
    "setuptools_scm>=8",
```